src/app/visual/split-view/split-view-visual.component.scss

```diff
@@ -0,0 +1,33 @@
+/*
+  The consumer has full control over what can be placed in the list panel.
+  These classes are simply for visual tests and not inteded to be used as a demo.
+*/
+.sky-visual-list {
+  margin: 0;
+  padding: 0;
+  list-style: none;
+
+  li {
+    padding: 10px 5px;
+    border-top:1px solid #ccc;
+  }
+}
+
+::ng-deep .sky-split-view-list  p {
+  padding-right:10px;
+  padding-left:10px;
+}
+
+::ng-deep .sky-split-view-workspace-content {
```

Comment:
If this is going to be used for code demo, we should demonstrate the preferred approach of wrapping your content in a div and applying e.g. `class="sky-padding-even-large"`

Reply:
Done.